app/services/levels/merge_service.rb

```diff
@@ -0,0 +1,33 @@
+module Levels
+  class MergeService
+    def initialize(level)
+      @level = level
+    end
+
+    def merge_into(other_level)
+      Level.transaction do
+        # Link startups and target groups to supplied level.
+        @level.startups.update_all(level_id: other_level.id) # rubocop:disable Rails/SkipsModelValidations
```

Comment:
@harigopal : Should we be allowing this operation if the `other_level` is level zero, which is generally not meant to be a level with students? There are features like the student report for coaches and the performance page for students, where this would result in a crash, as the applicable levels filter out level zero to show progress.

Reply:
A safer option for handling startups would be to keep them in its current level number if it exists after the merge operation.

Reply:
@mahesh-krishnakumar Good find. I've opted to just disallow merging into L0 to keep things simple. We'll let school admins weigh the consequences of shifting students into higher levels / jumping levels drastically.

Also, #141 should allow admins to avoid such issues by moving content out before merging levels in a conservative manner.